pkg/util/helper/binding.go

```diff
@@ -419,3 +412,35 @@ func GetResourceBindings(c client.Client, ls labels.Set) (*workv1alpha1.Resource
 
 	return bindings, c.List(context.TODO(), bindings, listOpt)
 }
+
+// GetWorksByLabels returns a WorkList by labels
+func GetWorksByLabels(c client.Client, ls labels.Set) (*workv1alpha1.WorkList, error) {
```

Comment:
Rename `GetWorksByLabels` to `GetWorks()` or also rename `GetClusterResourceBindings` and `GetResourceBindings` as well.